extensions/arc/src/ui/dashboards/miaa/miaaDashboardOverviewPage.ts

```diff
@@ -0,0 +1,154 @@
+import * as loc from '../../../localizedConstants';
+import { DashboardPage } from '../../components/dashboardPage';
+import { IconPathHelper } from '../../../constants';
+import { ControllerModel } from '../../../models/controllerModel';
+import { resourceTypeToDisplayName } from '../../../common/utils';
+
+export class MiaaDashboardOverviewPage extends DashboardPage {
+
+	private _arcResourcesTable!: azdata.DeclarativeTableComponent;
+
+	constructor(modelView: azdata.ModelView, private _controllerModel: ControllerModel) {
+		super(modelView);
+		this.refresh().catch(e => {
+			console.log(e);
+		});
+	}
+
+	public get title(): string {
+		return loc.overview;
+	}
+
+	public get id(): string {
+		return 'miaa-overview';
+	}
+
+	public get icon(): { dark: string, light: string } {
+		return IconPathHelper.properties;
+	}
+
+	protected async refresh(): Promise<void> {
+		await this._controllerModel.refresh();
+		this.eventuallyRunOnInitialized(() => this._arcResourcesTable.data = this._controllerModel.registrations().map(r => [r.instanceName, resourceTypeToDisplayName(r.instanceType), r.vCores]));
+	}
+
+	public get container(): azdata.Component {
+
+		const rootContainer = this.modelView.modelBuilder.flexContainer()
+			.withLayout({ flexFlow: 'column' })
+			.withProperties({ CSSStyles: { 'margin': '18px' } })
+			.component();
+
+		const propertiesContainer = this.modelView.modelBuilder.propertiesContainer().withProperties<azdata.PropertiesContainerComponentProperties>({
+			propertyItems: [
+				{
+					displayName: loc.resourceGroup,
+					value: 'contosoRG123'
+				},
+				{
+					displayName: loc.region,
+					value: 'West US'
+				},
+				{
+					displayName: loc.subscription,
+					value: 'contososub5678'
+				},
+				{
+					displayName: loc.subscriptionId,
+					value: '88abe223-c630-4f2c-8782-00bb5be874f6'
+				},
+				{
+					displayName: loc.state,
+					value: 'Connected'
+				},
+				{
+					displayName: loc.host,
+					value: 'plainscluster.sqlarcdm.database.windows.net'
+				}
+			]
+		}).component();
+
+		rootContainer.addItem(propertiesContainer);
+
+		const arcResourcesTitle = this.modelView.modelBuilder.text()
+			.withProperties<azdata.TextComponentProperties>({ value: loc.arcResources })
+			.component();
+
+		rootContainer.addItem(arcResourcesTitle, {
+			CSSStyles: {
+				'font-size': '14px'
+			}
+		});
+
+		this._arcResourcesTable = this.modelView.modelBuilder.declarativeTable().withProperties<azdata.DeclarativeTableProperties>({
+			data: [],
+			columns: [
+				{
+					displayName: loc.name,
+					valueType: azdata.DeclarativeDataType.string,
+					width: '33%',
+					isReadOnly: true
+				}, {
+					displayName: loc.type,
+					valueType: azdata.DeclarativeDataType.string,
+					width: '33%',
+					isReadOnly: true
+				}, {
+					displayName: loc.computeAndStorage,
+					valueType: azdata.DeclarativeDataType.string,
+					width: '34%',
+					isReadOnly: true
+				}
+			],
+			width: '100%',
```

Comment:
I put this 100% width on my tables too.  That was the only way I could get them to take the full width of the page like the designs.  But the declarative table component doesn't respect it.  I had to add [style.width]="getWidth()" to the table element in declarativeTable.component.ts.   Not a big deal just pointing it out.

Reply:
It looks like you didn't change this in master - were you intending on doing that at some point? It seems reasonable that it should respect the width property.